### PR TITLE
add frequency data to the NoiseLevelObserved model

### PR DESCRIPTION
**Pull request by thelxinoe**
Aims to add frequency data to the NoiseLevelObserved model.

### CODE REVIEW

File: specs/Environment/NoiseLevelObserved/doc/spec.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 It represents an observation of those acoustic parameters that estimate noise
 pressure levels at a certain place and time. This entity is primarily associated
 with the Smart City and environment vertical segments and related IoT
-applications.
+applications. In addition it also represents
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/doc/spec.md]: this sentence is not finished

File: specs/Environment/NoiseLevelObserved/doc/spec.md [hunk 2]
```diff
@@ -130,9 +137,16 @@ There are two options for representing them:
...
-**Note**: JSON Schemas are intended to capture the data type and associated
-constraints of the different Attributes, regardless their final representation
-format in NGSI(v2, LD).
+### Representing Frequency Data
+
+As the number of frequencies collected may vary from the ISO standard it is suggested that anything
+outside of this range be omitted and if frequencies within this range are missing the values are set
+to 0
+
+**Note**: JSON Schemas only capture the NGSI simplified representation, this
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/doc/spec.md]: this sentence should not be here

File: specs/Environment/NoiseLevelObserved/doc/spec.md [hunk 3]
```diff
@@ -211,62 +291,28 @@ Sample uses simplified representation for data consumers `?options=keyValues`
     "location": {
         "type": "Point",
         "coordinates": [-2.698, 42.8491]
-    }
-}
-```
-
-### LD Example
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/doc/spec.md]: be careful you are removing content.

File: specs/Environment/NoiseLevelObserved/doc/spec.md [hunk 4]
```diff
@@ -211,62 +291,28 @@ Sample uses simplified representation for data consumers `?options=keyValues`
...
-    },
-    "LAS": {
-        "type": "Property",
-        "value": 91.6
-    },
-    "@context": [
-        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
-        "https://schema.lab.fiware.org/ld/context"
-    ]
+    "frequency":{
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/doc/spec.md]: this is incorrect. please check. it should be ```"type": "Property", "value": <your structured value>```

File: specs/Environment/NoiseLevelObserved/example-normalized-ld.jsonld [hunk 5]
```diff
@@ -41,6 +41,87 @@
         "type": "Property",
         "value": 91.6
     },
+    "frequencies": {
+        "type": "object",
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/example-normalized-ld.jsonld]: be careful this is the NGSI-LD example "type": "Property"

File: specs/Environment/NoiseLevelObserved/schema.json [hunk 6]
```diff
@@ -33,6 +33,68 @@
             "0", "1", "2"
           ]
         },
+        "frequencies": {
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/schema.json]: the schema is wrong. please check consistency. is it frequency or frequencies?

File: specs/Environment/NoiseLevelObserved/schema.json [hunk 7]
```diff
@@ -33,6 +33,68 @@
...
+                "type": "integer"       
+            },
+            "8000" : {
+                "type": "integer"       
+            },
+            "10000" : {
+                "type": "integer"       
+            },
+        },
+        "type": "object"
```
Comment (jmcanterafonseca) [inline specs/Environment/NoiseLevelObserved/schema.json]: check the JSON Schema syntax

**Comment by jmcanterafonseca** [comment]
the CI is broken. you need to fix everything until CI is green

thanks for the effort!

**Comment by jmcanterafonseca** [comment]
@thelxinoe your PR has to be rebased. 

I'm now going on vacation, for any doubt please ask @jason-fox @emastrosimone or Luis Diez or Ignacio Martinez in the SynchroniCity project

**Comment by thelxinoe** [comment]
I think my recent commit has addressed all comments. Thanks.

**Comment by jmcanterafonseca** [comment]
it hasn't your PR does not merge it has to be rebased. that happens a lot

**Comment by jason-fox** [comment]
After the rebase, to ensure your files are formatted correctly

```
cd validator
npm i
npm run prettier
npm run prettier:text
```

should ensure that the JSON is formatted *consistently*.  The files will also be automatically prettified on commit.

**Comment by jmcanterafonseca** [comment]
we need to close this PR as soon as possible. The PR is totally wrong, please could you start over and provide just a patch with the needed changes on NoiseLevelObserved ?

@thelxinoe

**Comment by thelxinoe** [comment]
> we need to close this PR as soon as possible. The PR is totally wrong, please could you start over and provide just a patch with the needed changes on NoiseLevelObserved ?
> 
> @thelxinoe

Are you pushing to master? You made about 30 broken commits to the master branch on Aug 2, 2019 alone. Gonna be hard to hit that moving target.

**Pull request closed without merging**